golem/managers/demand/single_use.py

```diff
@@ -0,0 +1,102 @@
+import logging
+from itertools import chain
+from typing import Awaitable, Callable, Sequence
+
+from golem.managers.base import DemandManager
+from golem.managers.mixins import BackgroundLoopMixin
+from golem.node import GolemNode
+from golem.payload import Payload
+from golem.payload import defaults as payload_defaults
+from golem.resources import Allocation, Demand, Proposal
+from golem.resources.demand.demand_builder import DemandBuilder
+from golem.utils.asyncio import ErrorReportingQueue
+from golem.utils.logging import trace_span
+
+logger = logging.getLogger(__name__)
+
+
+class SingleUseDemandManager(BackgroundLoopMixin, DemandManager):
+    def __init__(
```

Comment:
It would be nice to have a pydoc that describes what is special about this one so it is easier to tell how it is different then other managers of the same type.